Description: Bump bounds for newer vector
 Bump bounds for vector 0.12.0.1, shipped in unstable
Author: Ilias Tsitsimpis <iliastsi@debian.org>
Forwarded: not-needed, fixed in newer version
Index: b/repa.cabal
===================================================================
--- a/repa.cabal
+++ b/repa.cabal
@@ -24,7 +24,7 @@ Library
         base                 >= 4.8 && < 4.10
       , template-haskell
       , ghc-prim
-      , vector               == 0.11.*
+      , vector               >= 0.11 && < 0.13
       , bytestring           == 0.10.*
       , QuickCheck           >= 2.8 && < 2.10
 
